style.ts

```diff
@@ -549,12 +553,22 @@ export interface ColorMap {
   extended?: boolean;
 }
 
+/**
+ * A VendorOption defines the algorithm to apply for Normalize contrast enhancement.
+ */
+export interface VendorOption {
+  algorithm?: 'StretchToMinimumMaximum' | 'ClipToMinimumMaximum' | 'ClipToZero';
+  minValue?: number;
+  maxValue?: number;
+}
```

Comment:
VendorOption seems a generic name but in this case is used only for ContrastEnhancement and it has a specific structure that match what the GeoServer expect.

Probably we should make this more specific:

1)
```js
export interface ContrastEnhancementVendorOption {
  algorithm?: 'StretchToMinimumMaximum' | 'ClipToMinimumMaximum' | 'ClipToZero';
  minValue?: number;
  maxValue?: number;
}
```

2)
```js
export interface ContrastEnhancementAlgorithm {
  name?: 'StretchToMinimumMaximum' | 'ClipToMinimumMaximum' | 'ClipToZero';
  minValue?: number;
  maxValue?: number;
}

export interface ContrastEnhancement {
  enhancementType?: 'normalize' | 'histogram';
  gammaValue?: number;
  algorithm?: ContrastEnhancementAlgorithm;
}
```

or more generic to be used in other context

3)
```js
export interface VendorProperty {
  name?: string;
  options?: any;
}
```

From the previous code it seems they prefer specific naming. My suggestions were more about the meaning of VendorOption itself that in this context seems to generic

Reply:
We agreed to change only the name of VendorOption to ContrastEnhancementVendorOption